tests/e2e/e2e_test.go

```diff
@@ -695,30 +643,12 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 	})
 })
 
-func waitForCustomChainsHealthy() string {
-	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
-	var created bool
-	continueLoop := true
-	for continueLoop {
-		select {
-		case <-ctx.Done():
-			continueLoop = false
-		case <-time.After(5 * time.Second):
-			cctx, ccancel := context.WithTimeout(context.Background(), 15*time.Second)
-			status, err := cli.Status(cctx)
-			ccancel()
-			gomega.Ω(err).Should(gomega.BeNil())
-			created = status.ClusterInfo.CustomChainsHealthy
-			if created {
-				continueLoop = false
-				existingSubnetID := status.ClusterInfo.GetSubnets()[0]
-				gomega.Ω(existingSubnetID).Should(gomega.Not(gomega.BeNil()))
-				cancel()
-				return existingSubnetID
-			}
-		}
-	}
+func getExistingSubnetID() string {
+	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+	status, err := cli.Status(ctx)
```

Comment:
This is also weird. I don't think `Status` should be blocking. If you want a Status, you want it right away from the client-side.

Then it's the client's job to wait until a certain Status is reached.